src/extension.ts

```diff
@@ -172,7 +173,9 @@ export function activate(context: vscode.ExtensionContext): void {
         }
 
         const createdApp = <IAzureNode<WebAppTreeItem>>await node.createChild(this);
-
+        const createdAppConfig: SiteConfigResource = await createdApp.treeItem.client.getSiteConfig();
```

Comment:
We don't want to await this just for telemetry. Do something like this:
https://github.com/Microsoft/vscode-azuretools/blob/master/appservice/src/deploy/deploy.ts#L33